_application_development/github/rebase.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Interactive Rebase
+position: 2
+---
+
+# Using rebase to ensure a clean commit history
```

Comment:
I use sourcetree's visual interactive rebase for this, but in general avoiding the need is a good idea.

If we have small enough feature branches and PRs then the squash and merge flow works really well.